--- basetsd.h	2011-09-18 18:40:35 +0100
+++ basetsd.h.new	2011-10-21 15:03:02 +0100
@@ -27,12 +27,12 @@
 extern "C" {
 #endif
 
-  typedef signed char INT8,*PINT8;
-  typedef signed short INT16,*PINT16;
+  //typedef signed char INT8,*PINT8;
+  //typedef signed short INT16,*PINT16;
   typedef signed int INT32,*PINT32;
   __MINGW_EXTENSION typedef signed __int64 INT64,*PINT64;
-  typedef unsigned char UINT8,*PUINT8;
-  typedef unsigned short UINT16,*PUINT16;
+  //typedef unsigned char UINT8,*PUINT8;
+  //typedef unsigned short UINT16,*PUINT16;
   typedef unsigned int UINT32,*PUINT32;
   __MINGW_EXTENSION typedef unsigned __int64 UINT64,*PUINT64;
   typedef signed int LONG32,*PLONG32;
